package.json

```diff
@@ -1,5 +1,15 @@
 {
     "name": "@goauthentik/authentik",
     "version": "2025.2.4",
-    "private": true
+    "private": true,
+    "type": "module",
+    "devDependencies": {
+        "@trivago/prettier-plugin-sort-imports": "^5.2.2",
+        "prettier": "^3.3.3",
+        "prettier-plugin-organize-imports": "^4.1.0",
+        "prettier-plugin-packagejson": "^2.5.10",
```

Comment:
Fixes Makefile's expectation of a global `prettier`, now using same module as other repo packages.